src/dev/build/tasks/os_packages/create_os_package_tasks.ts

```diff
@@ -172,3 +172,26 @@ export const CreateDockerContexts: Task = {
     });
   },
 };
+
+export const CreateDockerFIPS: Task = {
+  description: 'Create Docker Ubuntu and UBI images with FIPS',
+
+  async run(config, log, build) {
+    await runDockerGenerator(config, log, build, {
+      architecture: 'x64',
```

Comment:
Mind if we also add ARM builds?  I expect we'll want both eventually, but mostly now so I can run a test build.

Reply:
I will try to circle back to this or make a new issue for it. I ran a build using `aarch64` but the OpenSSL compilation step was running for ~35 minutes before I cancelled it. From a bit of research, it looks like we will have to compile OpenSSL differently for ARM as well.